config/src/main/resources/org/springframework/security/config/spring-security-6.0.xsd

```diff
@@ -1277,7 +1277,7 @@
       <xs:attribute name="security-context-explicit-save" type="xs:boolean">
          <xs:annotation>
             <xs:documentation>Optional attribute that specifies that the SecurityContext should require explicit saving
-                rather than being synchronized from the SecurityContextHolder. Defaults to "false".
+                rather than being synchronized from the SecurityContextHolder. Defaults to "true".
```

Comment:
It came with a rebase...

Reply:
Gotcha. You may need to revert this change on your branch and squash the commit, so your diff does not include this change. Then I think you should be able to rebase again.

Reply:
I don't know exactly how, but I think I picked a version from main just before a regression: because of gh-11762, default value became "true". This can be seen in `spring-security-6.0.rnc` and more importantly in `HttpConfigurationBuilder.java` line 326: `return !StringUtils.hasText(explicitSaveAttr) || Boolean.parseBoolean(explicitSaveAttr);`

Do you really prefer I remove this change from this PR and create an issue just for this documentation line?

Reply:
I will revert to false, but rnc and xsd won't be in synch. I already squashed and commited.

Reply:
Created a separate pull request: https://github.com/spring-projects/spring-security/pull/11773

Reply:
Ok, sorry I didn't realize they were out of sync.